application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -102,6 +121,31 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             .insert());
     }
 
+    private void handleListCommand(SlashCommandInteractionEvent event) {
+        BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
+                Channel: %s
+                Remind at: %s""".formatted(MessageUtils.mentionChannelById(channelId),
+                TimeFormat.DEFAULT.format(remindAt));
+
+        EmbedBuilder remindersEmbed = new EmbedBuilder().setTitle("Pending reminders")
+            .setColor(RemindRoutine.AMBIENT_COLOR);
+
+        database
+            .read(context -> context.selectFrom(PENDING_REMINDERS)
+                .where(PENDING_REMINDERS.GUILD_ID.eq(event.getGuild().getIdLong())
+                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(event.getUser().getIdLong())))
```

Comment:
Using `.and` makes it unneccesairily unreadable. You can just pass all of the requirements to `.where`
Example `.where(requirement1, requirement2)` :+1:

Reply:
i believe .and makes it more readable and understandable

Reply:
But then at least thore the guild and user id in a variable to make the lines shorter